src/app/components/profile/profile.html

```diff
@@ -214,9 +217,11 @@
                                     <div class="row">
                                         <div class="col-md-12">
                                             <div class="name">
-                                                {{ skill.name }}
+                                                {{ skill.name }} 
+                                                <p id= "expertise" ng-show= {{skill.expertise}} class="fa fa-star"</p> 
+                                                <p id= "experience" ng-show= {{skill.experience}} class="fa fa-arrow-up"</p> 
+                                            </div>
```

Comment:
wrong indentation, and there is a div with no class on row 209, please remove that.